src/backend/deep_search/views.py

```diff
@@ -0,0 +1,283 @@
+"""
+Albert chatbot integration views.
+
+This module provides the core API endpoints for the chatbot functionality.
+"""
+
+import logging
+from typing import Dict, Any
+
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework import status
+
+from .chatbot import get_chatbot
+from .email_service import EmailService
+
+logger = logging.getLogger(__name__)
+
+
+@api_view(['POST'])
```

Comment:
IMO this should be a get endpoint not a post as it does not create resource then the query can be easily get from a query param